ironfish-cli/src/commands/wallet/multisig/ledger/backup.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { IronfishCommand } from '../../../../command'
+import { Ledger } from '../../../../utils/ledger'
+
+export class MultisigLedgerBackup extends IronfishCommand {
+  static description = `Backup encrypted multisig keys from a Ledger device`
```

Comment:
Trying to think of a way to make this more in-line with the new description format. Would something like `show encrypted multisig key from a Ledger device` be accurate? As far as I can tell, the "backup" is really "log to user", if I'm understanding this correctly.